config/routes.rb

```diff
@@ -1,12 +1,15 @@
 Ichabod::Application.routes.draw do
+  resources :collections
+
   resources :nyucores
 
   root :to => "catalog#index"
   Blacklight.add_routes(self)
 
+  get 'collections', :to => "collections#index"
```

Comment:
Kate, I don't think this is needed.  The `resources :collections` statement on line 2 includes this case.